components/overflow-group/filter-overflow-group.js

```diff
@@ -0,0 +1,62 @@
+import '../filter/filter.js';
+import { css, html, LitElement } from 'lit';
+import { OVERFLOW_DROPDOWN_CLASS, OverflowGroupMixin } from './overflow-group-mixin.js';
+
+function createFilterItem(node) {
+	const dimensionSets = node.querySelectorAll('d2l-filter-dimension-set');
+	const clones = Array.from(dimensionSets).map((set) => set.cloneNode(true));
+	return clones;
+}
+
+/**
+ * A component that can be used to display a group of filters that will be put into an overflow filter when they no longer fit on the first line of their container
+ * @slot - d2l-filters to be added to the container
+*/
+class OverflowGroupFilter extends OverflowGroupMixin(LitElement) {
```

Comment:
This file will be deleted before merge and is to just show an example of how this would work.

Consumers of `OverflowGroupMixin` would need to implement `convertToOverflowItem` (specifies what a chomped item becomes in the overflow "dropdown") and `getOverflowMenu` (specifies the chomp overflow menu)